pages/onboarding.jsx

```diff
@@ -83,6 +84,19 @@ const Onboarding = () => {
     }
   };
 
+  const validateTeam = (value) => {
+    const equipmentIdValue = parseInt(value);
+     if( !isNaN(equipmentIdValue) ) {
+      if(equipmentIdValue % 2 !== 0) {
```

Comment:
You can add `&& codename` to these two conditions to as some safety. In theory all three fields must be non-blank to be submittable. There a few ways to enforce this. I need to dive into this more because there are some behaviors with useState that make this weird depending on the approach.

Reply:
Here is a slightly jank way of doing the for which buttons should be disabled onChange for codename field. You could extract this into it's own function. There is some trouble with referencing codename after setCodename and it returning the old value, so this implementation does it in a "safe" way.
```javascript
onChange={(e) => {
  const value = e.target.value;
  if (value === "" || equipmentId === "") {
    setAddGreenButtonDisabled(true);
    setAddRedButtonDisabled(true);
  } else if (equipmentId % 2 === 0) {
    setAddGreenButtonDisabled(false);
    setAddRedButtonDisabled(true);
  } else if (equipmentId % 2 !== 0) {
    setAddRedButtonDisabled(false);
    setAddGreenButtonDisabled(true);
  }
  setCodename(value);
}}
```